packages/ai-native/src/browser/components/ChangeList.tsx

```
@@ -0,0 +1,105 @@
+  onFileClick: (path: string) => void;
+}
+
+export const FileListDisplay: React.FC<FileListDisplayProps> = ({ files, onFileClick }) => {
+  const [isExpanded, setIsExpanded] = useState(true);
+  const labelService = useInjectable<LabelService>(LabelService);
+  const fileIcons = useMemo(
+    () =>
+      files.map((file) => {
+        const uri = URI.parse(file.path);
+        const iconClass = labelService.getIcon(uri);
+        return <span className={iconClass}></span>;
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)